### PR TITLE
Add -A option to StringTie

### PR DESCRIPTION
**Pull request by susannasiebert**
This will output a tsv file with gene expression data.

### CODE REVIEW

File: rnaseq/stringtie.cwl [hunk 1]
```diff
@@ -31,4 +32,8 @@ outputs:
     gtf:
         type: File
         outputBinding:
-            glob: transcripts.gtf
+            glob: stringtie_transcripts.gtf
+    tsv:
```
Comment (tmooney) [inline rnaseq/stringtie.cwl]: `gtf` is good inspiration for this, but `tsv` is a rather generic name! Maybe `gene_expression` (or `gene_expression_tsv`)?  (I'd probably rename the other one too!)

Comment (susannasiebert) [reply rnaseq/stringtie.cwl]: Yeah, you're right. I went ahead and renamed all of these labels to something more descriptive.

**Comment by chrisamiller** [comment]
+1  If you point me to a checkout with this change, I can test it on the small test bams. (or just run it yourself using `/gscmnt/gc2708/info/medseq/tmp/toiltest/rnatest.yaml`

**Comment by susannasiebert** [comment]
@chrisamiller try `/gscmnt/gc2764/cad/ssiebert/toil_test/arvados_trial`.

**Comment by chrisamiller** [comment]
works fine - output here looks sane: `/gscmnt/gc2708/info/medseq/tmp/toiltest/output`

**Comment by susannasiebert** [comment]
Great! Thanks for testing. I'll merge.

**Pull request merged**